### ISSUE TITLE
Cookie-based routing in Ingress specification (canary deployment)

### ISSUE THREAD

**Issue by antonmatsiuk** [jcmoraisjr/haproxy-ingress#390]
Is it possible to define cookie-based routing to a particular backend in the Ingress specification? 
In a traditional HAProxy config it's possible via specifying an ACL in the frontend config, e.g.:
```
frontend frontend-X
  acl is_cookie_set hdr_sub(cookie) COOKIE
  use_backend backend-Y if is_cookie_set
```
according to description [here](https://github.com/jcmoraisjr/haproxy-ingress#configuration-snippet) it should be possible to do globally with `config-frontend:` but not for a specific Ingress. For example in nginx-ingress it's supported via [canary annotation](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#canary)

**Comment by jcmoraisjr** [comment]
Hi, cookie-based routing is currently unsupported but already in the short term backlog.

**Comment by jcmoraisjr** [comment]
Closing due do duplication with #254 . Btw it was just implemented, v0.9 snapshot next week.
